public/icons/chevron_up_down.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="32px" height="32px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Artboard" fill="#000000" fill-rule="nonzero">
+            <g id="Group" transform="translate(4.000000, 0.000000)">
+                <g id="Path" transform="translate(0.000000, 17.777778)">
+                    <polygon points="11.3777778 14.049341 0 2.67156323 2.06935162 0.609523812 11.3777778 9.9106378 20.6862039 0.609523812 22.7555556 2.67156323"></polygon>
+                </g>
+                <g id="Path" transform="translate(11.377778, 7.111111) rotate(-180.000000) translate(-11.377778, -7.111111) translate(0.000000, 0.000000)">
+                    <polygon points="11.3777778 14.049341 0 2.67156323 2.06935162 0.609523812 11.3777778 9.9106378 20.6862039 0.609523812 22.7555556 2.67156323"></polygon>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
Could be minimized https://www.svgminify.com/

Reply:
Good point, done